app/views/admins/orders/_form_search.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= form_tag admins_orders_path, method: :get,class: "form-search-admin" do %>
```

Comment:
dùng form khác được kh em, có cần `method: :get` không

Reply:
không được anh ơi , vì form for thì cần object , dùng để create object còn method thì bỏ được anh à =))

Reply:
bỏ method get dc k

Reply:
method bỏ được c ạ

Reply:
method không bỏ được ạ, default là method post c ạ